<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="6.578">
 <testsuite name="Demo Calculator Test" timestamp="2020-09-17T21:07:42" hostname="localhost" time="6.578" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="Demo Calculator Test" name="Addition Test" time="6.161" />
  <testcase classname="Demo Calculator Test" name="Subtraction Test" time="0.41">
   <failure type="exception" message="Failed: hp.clickGo1 is not a function"><![CDATA[TypeError: hp.clickGo1 is not a function
    at UserContext.<anonymous> (C:\Users\swetha2\Desktop\Projects\ProtractorDemo\tests\calculator.js:41:12)
    at C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: Run it("Subtraction Test") in control flow
    at UserContext.<anonymous> (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\swetha2\Desktop\Projects\ProtractorDemo\tests\calculator.js:29:5)
    at addSpecsToSuite (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\swetha2\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\swetha2\Desktop\Projects\ProtractorDemo\tests\calculator.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>